UI/EmuScreen.cpp

```diff
@@ -356,6 +357,11 @@ void EmuScreen::sendMessage(const char *message, const char *value) {
 		} else {
 			gstate_c.skipDrawReason &= ~SKIPDRAW_WINDOW_MINIMIZED;
 		}
+	} else if (!strcmp(message, "chat screen")) {
+		releaseButtons();
+		ChatMenu * ch = new ChatMenu();
```

Comment:
What if I press the button while there's already a chat menu?  Does it make more sense to just keep the existing chat menu?

-[Unknown]